### DISCUSSION TITLE
MetaQA-half

### DISCUSSION THREAD

**Discussion by lxmm0427** [jishnujayakumar/MLRC2020-EmbedKGQA#14] [Q&A]
Hi,
I  get 0.68 in metaQA half KG-hop1. I'm using your command with the same hyperparameter setting. After the first epoch, I get 0.68, and then the score stays the same. After 10 epochs, the training automatically ends.0.68 is too low, I do not know what's wrong with it?  I would like to ask whether you get the result 83.9 in the paper?

**Comment by jishnujayakumar** [comment]
The relation matching (RM) module is the key. If you include that and run for larger epochs, I think you would get better results. RM module can be found in the original paper's repository. I hope that this clears your doubt. Also, a hyperparameter search could also help to filter out the useful ones.

**Reply by lxmm0427** [reply]
Thanks for your reply, I will try it.
